src/Umbraco.Web.BackOffice/Controllers/EntityController.cs

```diff
@@ -568,7 +568,7 @@ public ActionResult<UrlAndAnchors> GetUrlAndAnchors(Udi id, string culture = "*"
     [HttpGet]
     public UrlAndAnchors GetUrlAndAnchors(int id, string? culture = "*")
     {
-        culture ??= ClientCulture();
+        culture = culture is null or "*" ? ClientCulture() : culture;
```

Comment:
This looks weird... but it was the only way I could get around it without making breaking changes.